core/templates/dev/head/components/concept-card/concept-card.template.html

```diff
@@ -1,17 +1,17 @@
 <div class="oppia-concept-card-viewer" ng-if="!$ctrl.loadingMessage">
   <div class="concept-card-explanation">
-    <b> Explanation </b>
+    <b translate="I18N_CONCEPT_CARD_EXPLANATION"></b>
     <div style="margin-bottom: 10px"> <[$ctrl.getSkillExplanation()]> </div>
   </div>
   <div class="concept-card-worked-examples">
-    <b> Worked Examples </b>
+    <b translate="I18N_CONCEPT_CARD_WORKED_EXAMPLES"></b>
     <div ng-repeat="workedExample in $ctrl.showWorkedExamples()">
       <div> <[workedExample]> </div>
     </div>
     <button class="btn btn-secondary show-more-worked-examples-button"
             ng-if="!$ctrl.isLastWorkedExample()"
-            ng-click="$ctrl.showMoreWorkedExamples()">
-      + Worked Examples
```

Comment:
If this is to show more worked examples, might be better to say "show more"? "+ Worked examples" might not be very clear IMO.